dev/io.openliberty.microprofile.metrics.5.0.monitor.internal/src/io/openliberty/microprofile/metrics/internal/monitor/internal/MonitorMetrics.java

```diff
@@ -261,6 +261,10 @@ public void unregisterMetrics(SharedMetricRegistries sharedMetricRegistry) {
         }
         vendorMetricIDs.clear();
     }
+    
+    public Set<MetricID> getMetricIDSet() {
```

Comment:
Better to rename it to `getVendorMetricIDSet` to avoid confusion.

Reply:
Done.